auth/src/main/java/com/firebase/ui/auth/ui/HelperActivityBase.java

```diff
@@ -1,30 +1,87 @@
 package com.firebase.ui.auth.ui;
 
+import android.app.Activity;
+import android.content.Context;
 import android.content.Intent;
 import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
 import android.support.annotation.RestrictTo;
 import android.support.v7.app.AppCompatActivity;
 
+import com.firebase.ui.auth.IdpResponse;
+import com.firebase.ui.auth.ResultCodes;
+import com.firebase.ui.auth.util.signincontainer.SaveSmartLock;
+import com.google.firebase.auth.FirebaseUser;
+
+import static com.firebase.ui.auth.util.Preconditions.checkNotNull;
+
 @SuppressWarnings("Registered")
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 public class HelperActivityBase extends AppCompatActivity {
 
-    protected ActivityHelper mActivityHelper;
+    protected FlowParameters mFlowParameters;
+    protected ProgressDialogHolder mProgressDialogHolder;
```

Comment:
It looks like the dialog can also be private since you have a getter. Or we could remove the getter, seems easier and would be more consistent with the other code you've written.